blockchain/fullblocktests/generate.go

```diff
@@ -1066,7 +1065,6 @@ func Generate() (tests [][]TestInstance, err error) {
 	//   ... -> b39(10)
 	//                \-> b40(11)
 	//
-	tooManySigOps = repeatOpcode(txscript.OP_CHECKMULTISIGVERIFY, maxBlockSigOps/20)
```

Comment:
Same here as above.  I think it would be less error prone to keep this line and change the line below to append to it.